assets/src/styles/blocks/core-overrides/Table.scss

```diff
@@ -4,44 +4,45 @@
   table {
     // Grey background (defaut)
     th {
-      background-color: $table-header-grey;
+      background-color: var(--table-header-grey--background, $table-header-grey);
     }
 
     tr {
-      background-color: $table-even-row-grey;
+      background-color: var(--table-even-row-grey--background, $table-even-row-grey);
     }
 
     tfoot td {
-      background-color: $table-footer-grey;
+      background-color: var(--table-footer-grey--background, $table-footer-grey);
+      font-weight: bold;
```

Comment:
If we change the footer color, do we still want the bold font weight too?

Reply:
He approved it, so I think so.